package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typed-rest-client",
-  "version": "1.7.1",
+  "version": "1.8.0",
```

Comment:
2 nits:
1) Since this is a bug fix, lets just update the patch version (to `1.7.2`)
2) The version should get updated in package-lock as well (I think this should just happen when you run npm install)

Reply:
Sure